Menus/ToolsMenu/ToolsMenuButton/ToolsMenuButton.cs

```diff
@@ -519,6 +519,10 @@ void OnBackgroundButtonClick()
 
 		void OnSecondaryButtonClicked()
 		{
+			if (!implementsSecondaryButton)
+			{
+				return;
+			}
```

Comment:
Lose brackets for single-line if

Reply:
CSharp standards reference says brackets for single-line ifs are optional; neither required nor prohibited

Reply:
Not required, but we've been doing this consistently throughout the codebase. The only exception is where an else has more than one line. Personally I'm in favor of consistency, and generally if you can eliminate lines of code, you can see more on screen. Especially in the case of an early-out, I'd say no brackets for sure.